core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -225,9 +228,12 @@ public static String getAppVersion() {
         return appVersion;
     }
 
+    @SuppressWarnings({"UnusedReturnValue", "WeakerAccess"})
     public static final class Options {
 
-        private String configFileName = "mobile-services.json";
+        public static final String DEFAULT_CONFIG_FILE_NAME = "mobile-services.json";
```

Comment:
perhaps move up? our of the nested `Options` ?

Reply:
Yes it would be better out. In `MobileCore`? Or some other class, maybe `MobileCoreContants` or something like that. WDYT?

Reply:
I think moving it to `MobileCore` is fine. Just to keep all the public static's at the same place.